pkg/gateway/fabric/flag.go

```diff
@@ -27,31 +26,11 @@ func (fn FlagName) String() string {
 }
 
 const (
-	// FlagNameNodeName is the name of the node.
-	FlagNameNodeName FlagName = "node-name"
-
 	// FlagNameEnableARP is the flag to enable ARP.
 	FlagNameEnableARP FlagName = "enable-arp"
 )
 
-// RequiredFlags contains the list of the mandatory flags.
-var RequiredFlags = []FlagName{
-	FlagNameNodeName,
-}
-
 // InitFlags initializes the flags for the gateway.
 func InitFlags(flagset *pflag.FlagSet, opts *Options) {
-	flagset.StringVar(&opts.NodeName, FlagNameNodeName.String(), "", "Name of the node")
```

Comment:
Do we need to remove this flag from templates?

Reply:
No, because it has been moved here https://github.com/cheina97/liqo/blob/512745154f7bd9d894929a647a6584885b7bccb0/pkg/gateway/flags.go#L77